ql/time/calendars/southkorea.cpp

```diff
@@ -66,6 +66,8 @@ namespace QuantLib {
             || (d == 1 && m == May)
             // Children's Day
             || (d == 5 && m == May)
+            || (d == 7 && m == May && y == 2018)
```

Comment:
Does this mean it's May 5th, but moves to the next Monday if it's on a Saturday or Sunday?  Did it always do that, or just these two years?

Reply:
I believe it was only for 2018.